docs/src/components/ButtonBoxList/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import styles from './styles.module.css';
+
+const ButtonBoxList = ({children}) => {
+  return (<ul className={styles.buttonBoxList}>
+  {React.Children.map(children, c  => <li className={styles.buttonBoxListItem}>{c}</li>)}
+  </ul>);
```

Comment:
@pascal-airy this shiny new component was brought to you with love by me and @AitorAlgorta 🙏